.github/workflows/ci.yml

```diff
@@ -18,8 +18,10 @@ jobs:
         python: ['3.10', '3.11', '3.12']
         django: ['4.2']
         mozilla_django_oidc: ['4.0']
+        setup_config_enabled: ['no', 'yes']
```

Comment:
can you also set up codecov flags for this? See https://github.com/maykinmedia/django-digid-eherkenning/blob/master/.github/workflows/ci.yml#L61 for a reference

Reply:
Added